gordon_gcp/schemas/event.schema.json

```diff
@@ -1,3 +1,49 @@
 {
-    "foo": "bar"
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "id": "https://github.com/spotify/gordon-gcp/blob/master/gordon_gcp/schemas/event.schema.json",
+    "title": "Generic Event Messages",
+    "description": "Schema for parsing generic event messages",
+    "required": ["name", "action", "type"],
+    "properties": {
+        "name": {
+            "description": "Record name",
+            "type": "string",
+            "examples": [
+                "www.example.com.",
+                "some-fqdn.subdomain.example.com."
+            ]
+        },
+        "rrdatas": {
```

Comment:
I'm thinking technically this isn't needed if it's a `deletions` action. I can't quite [discern](https://cloud.google.com/dns/api/v1/changes#resource) if it's required for deleting. If it is, we can adjust the schema, or first call the DNS API to list the record sets. Thoughts?

Reply:
Ok I'll add `rrdatas` as required. 

At least with TTL we can default it to 300 seconds if not given when we're actually processing the event.